edkahara/iReporter
# [finishes #162407549] create a user login page

__What does this PR do?__
Create a user login api endpoint

__Description of tasks to be completed.__
Create an api endpoint that facilitates the logging in of an existing user. Test the api endpoint.

__How should it be manually tested?__
Clone the project. Check out the ft-create-user-login-api-endpoint-#162407549 branch. Start the environment using env\Scripts\activate. Run flask run on the terminal and go to Postman. To perform the tests run nosetest.

__Relevant pivotal tracker stories.__
#162407549
